### PR TITLE
Feat: Add doctrine/orm v3 compatibility

### PR DESCRIPTION
**Pull request by bpolaszek**
Add doctrine/orm: ^3.0 compatibility.

### CODE REVIEW

**Review by norkunas — APPROVED** [review]
Just fix CS :)

**Comment by bpolaszek** [comment]
Done, ready for review 🙂

File: phpstan.neon.dist [hunk 1]
```diff
@@ -5,3 +5,5 @@ parameters:
 	paths:
 		- src
 		- tests
+	ignoreErrors:
```
Comment (norkunas) [inline phpstan.neon.dist:8]: Add this to baseline instead :)

Comment (bpolaszek) [reply phpstan.neon.dist:8]: What do you mean?

Comment (norkunas) [reply phpstan.neon.dist:8]: I meant separate baseline file.. but nevermind

**Comment by norkunas** [comment]
bors merge

**Comment by norkunas** [comment]
Thank you @bpolaszek

**Pull request merged**